docs/flox-5-minutes.md

```diff
@@ -5,637 +5,249 @@ description: Get started with creating Flox environments.
 
 # Flox in 5 minutes
 
-Flox is a virtual environment and package manager all in one.
-With Flox you create development environments that put you into reproducible
-subshells with dependencies provided and configured for you.
-Even better, these environments layer so you can prepare different environments
-for different contexts and stack them when needing to work across contexts.
+Flox is a next-generation, language-agnostic package and environment manager.
+With it you create sets of tools, environment variables, and setup scripts that work reproducibly from machine to machine, x86_64 to Arm, and macOS to Linux.
```

Comment:
Let's keep "stack" in this paragraph so that the next paragraph flows better.

```suggestion
With it you create stacks of tools, environment variables, and setup scripts that work reproducibly from machine to machine, x86_64 to Arm, and macOS to Linux.
```